GreenGarbageApp/app/src/main/res/layout/fragment_start.xml

```diff
@@ -13,15 +13,6 @@
         android:orientation="vertical"
         android:paddingTop="150dp">
 
-        <ImageView
```

Comment:
👍 We have discussed this. 
Im fine with removing this to focus on the log in process.